src/components/performance/export.cjsx

```diff
@@ -39,10 +42,15 @@ PerformanceExport = React.createClass
 
       exportState =
         downloadUrl: lastExport.url
-        lastExported: lastExport.created_at
+        lastExported: moment(lastExport.created_at).fromNow()
```

Comment:
Initially that's what I attempted.  However, it gets confused because the initial export call to the server sets time at the request time from server, and the created_at is always a few seconds ** after ** that as opposed to before.